lambda-function/src/utils/library/logging.ts

```diff
@@ -0,0 +1,23 @@
+import chalk from "chalk";
+
+export default class Logging {
+  // public static log = (args:any)=> this.info(args);
```

Comment:
Commented out duplicate line.

Reply:
That must have changed because of running "format". I didn't make any intentional changes to that file